docs/setup/pcsx2-ex-setup/pcsx2-ex-setup.md

```diff
@@ -90,7 +90,7 @@ Open the KH2FM Randomizer Seed Generator executable file (for this guide, its `C
 
 ![Seed Generator Window](./randomizer.png)
 
-On the Seed Generator Window, it’s time to create a seed. There are way too many things to tweak around in making a seed that I might’ve made a novel if I enumerated it all here one-by-one. To check what each setting does, hover your mouse on that area and read the tooltip that shows up.
+On the Seed Generator Window, it’s time to create a seed. There are way too many things to tweak around in making a seed that I might’ve made a novel if I enumerated it all here one-by-one, so [here's](https://tommadness.github.io/KH2Randomizer/seed-generator/) the round-up of all the seed generator's features. Otherwise, to check what each setting does, hover your mouse on that area and read the tooltip that shows up.
```

Comment:
Preferably use relative links for anything that stays within the site, in case the domain changes for whatever reason (I have no plans, but just futureproofing). In this case, `../../seed-generator` would be the appropriate link.